sql/core/src/test/scala/org/apache/spark/sql/execution/arrow/ArrowConvertersSuite.scala

```diff
@@ -1324,7 +1324,7 @@ class ArrowConvertersSuite extends SharedSQLContext with BeforeAndAfterAll {
       InternalRow(i)
     } :+ InternalRow(null)
 
-    val schema = StructType(Seq(StructField("int", IntegerType, nullable = true)))
```

Comment:
For completeness, let's just not make these changes

Reply:
OMG, I omitted it. Sure, reverted with another change.